airbyte-webapp/orval.config.ts

```diff
@@ -22,25 +22,25 @@ export default defineConfig({
       },
     },
   },
-  connectorBuilder: {
-    input: "../connector-builder-server/src/main/openapi/openapi.yaml",
-    output: {
-      target: "./src/core/request/ConnectorBuilderClient.ts",
-      prettier: true,
-      override: {
-        header: (info) => [
-          `eslint-disable`,
-          `Generated by orval 🍺`,
-          `Do not edit manually. Run "npm run generate-client" instead.`,
-          ...(info.title ? [info.title] : []),
-          ...(info.description ? [info.description] : []),
-          ...(info.version ? [`OpenAPI spec version: ${info.version}`] : []),
-        ],
-        mutator: {
-          path: "./src/core/request/apiOverride.ts",
-          name: "apiOverride",
-        },
-      },
-    },
-  },
+  #connectorBuilder: {
+  #  input: "../airbyte-connector-builder/src/main/openapi/openapi.yaml",
+  #  output: {
+  #    target: "./src/core/request/ConnectorBuilderClient.ts",
+  #    prettier: true,
+  #    override: {
+  #      header: (info) => [
+  #        `eslint-disable`,
+  #        `Generated by orval 🍺`,
+  #        `Do not edit manually. Run "npm run generate-client" instead.`,
+  #        ...(info.title ? [info.title] : []),
+  #        ...(info.description ? [info.description] : []),
+  #        ...(info.version ? [`OpenAPI spec version: ${info.version}`] : []),
+  #      ],
+  #      mutator: {
+  #        path: "./src/core/request/apiOverride.ts",
+  #        name: "apiOverride",
+  #      },
+  #    },
+  #  },
+  #},
```

Comment:
Undo the changes to this file - I need to have the typescript client generated now, since I am using those types even though I'm not actually calling the server